libs/pages/login/src/lib/hooks/use-redirect-if-logged/use-redirect-if-logged.tsx

```diff
@@ -75,7 +73,7 @@ export function useRedirectIfLogged() {
 
       fetchData()
     }
-  }, [navigate, checkIsAuthenticated, createAuthCookies, dispatch, userSignUp.dx_auth])
+  }, [navigate, checkIsAuthenticated, createAuthCookies, dispatch, sendDataToGTM, user])
```

Comment:
```suggestion
  }, [navigate, checkIsAuthenticated, createAuthCookies, dispatch, sendDataToGTM, user?.email])
```

Reply:
The hook need all `user` object 

<img width="1050" alt="Capture d’écran 2023-08-28 à 20 26 23" src="https://github.com/Qovery/console/assets/533928/0edab62a-777f-40da-9e4b-17751b3e8b19">